Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -232,7 +374,18 @@ s32 SDIOSlot0::ExecuteCommand(const Request& request, u32 _BufferIn, u32 _Buffer
   case ACMD_SENDOPCOND:
     // Sends host capacity support information (HCS) and asks the accessed card to send
     // its operating condition register (OCR) content
-    Memory::Write_U32(0x80ff8000, _BufferOut);
+    {
```

Comment:
Can you call `SDIOSlot0::GetOCRegister` instead, or at least a commonly used/extracted method instead of duplicating this piece of code?